cmd/apply.go

```diff
@@ -36,6 +37,11 @@ var applyCmd = &cobra.Command{
 			return e
 		}
 
+		oauth, e := cmd.Flags().GetBool("oauth")
```

Comment:
--oauth seems a bit ambiguous here. What do you think about not specifying a flag at all and instead having this behavior totally driven by the presence/absence of the env variable?